infrastructure/src/main/java/org/corfudb/infrastructure/log/statetransfer/transferprocessor/ParallelTransferProcessor.java

```diff
@@ -32,10 +32,23 @@
     /**
      * A number of in-flight requests per one node.
      */
-    private static final int NUM_REQUESTS_PER_NODE = 5;
+    private final int batchesPerNode;
```

Comment:
This not really a batch, it's the number of concurrent requests, we need to change the name.

Reply:
fixed